pkg/features/kube_features.go

```diff
@@ -99,6 +99,11 @@ const (
 	// Ability to Expand persistent volumes
 	ExpandPersistentVolumes utilfeature.Feature = "ExpandPersistentVolumes"
 
+	// owner: @mlmhl
+	// alpha: v1.11
+	// Ability to expand online persistent volumes
+	ExpandOnlinePersistentVolumes utilfeature.Feature = "ExpandOnlinePersistentVolumes"
```

Comment:
nit: How about `ExpandPersistentVolumesFSWithoutUnmounting`?

Since "online" is an overloaded term, and not immediately obvious what it means, I'd prefer the flag, comments, and documentation to explicitly refer to FS resizing without unmount for clarity.

Reply:
Done

Reply:
`ExpandPersistentVolumesFSWithoutUnmounting` is not quite accurate I think because even without this feature, we have been expanding mounted volumes, because certain file system types like `XFS` requires volume to be mounted for resizing to happen. 

How about `ExpandInUseVolumes` ?